### PR TITLE
Anh em tao dùng Visafe

### PR DESCRIPTION
**Pull request by tohruniverse**
- You should not worry about redirect to Adguard server anymore. After many hours of hard work, I fixed it.

### CODE REVIEW

**Comment by thangluu93** [comment]
LOL =)))))

**Comment by danglequocbao001** [comment]
Nice bait

**Comment by kietphan** [comment]
approved

**Comment by hieuht-2055** [comment]
Nice

**Comment by com-network** [comment]
> 
> 
>     * You should not worry about redirect to Adguard server anymore. After many hours of hard work, I fixed it.

Hey, DM me on know how you could search and replace text within so much files like that. I'd like to rip off software made by other people too, but I don't want it to be found out. Thanks a bunch 🤗

**Comment by BrainIs404** [comment]
ông ơi các file có tên Adguard chưa đổi kìa :)))

**Comment by Amethystafyy282** [comment]
> ông ơi các file có tên Adguard chưa đổi kìa :)))

  Nhanh trí tạo PR để kiếm contribution

**Comment by canhld94** [comment]
Nice work, I hope the team will merge this PR soon enough, it's magnificent. 
Just one small comment, don't just add a new commit like this but prefer squashing all previous commits. It better fits the team's engineering culture.

**Comment by AlphaNecron** [comment]
> > 
> > 
> >     * You should not worry about redirect to Adguard server anymore. After many hours of hard work, I fixed it.
> 
> Hey, DM me on how you could search and replace text within so much files like that. I'd like to rip off software made by other people too, but I don't want it to be found out. Thanks a bunch 🤗

VSCode has a function named `Search in files`, you can use it to replace words across files.

**Comment by AlphaNecron** [comment]
Nice "hard work", abusive of bulk replace without awareness.

**Comment by hieu-van** [comment]
> Nice "hard work", **_abusive_** of bulk replace without awareness. 

Lol. Are u trying to be serious

**Comment by AlphaNecron** [comment]
> > Nice "hard work", **_abusive_** of bulk replace without awareness. 
> 
> Lol. Are u trying to be serious

He was using bulk replace without noticing the URLs.

**Comment by QuanTrieuPCYT** [comment]
LMFAO

**Comment by EndermanHackerVN** [comment]
Hey can you teach me how to skid soft wear like vietnamese gov?

**Comment by ShikiHTM** [comment]
LOL

**Comment by nhathuy13598** [comment]
LGTM

**Comment by gdhaison** [comment]
CLGT, merge!

**Comment by phuclinhsama** [comment]
This

**Comment by AlphaNecron** [comment]
> This

Nguyen Thi Phuc Linh...

**Comment by huanlv94** [comment]
LGTM

File: AdguardExtension/Adguard-Config.xcconfig [hunk 1]
```diff
@@ -1,33 +1,33 @@
 //
-//   This file is part of Adguard for iOS (https://github.com/AdguardTeam/AdguardForiOS).
-//   Copyright © Adguard Software Limited. All rights reserved.
+//   This file is part of Visafe for iOS (https://github.com/VisafeTeam/VisafeForiOS).
```
Comment (vanthoainguyen) [inline AdguardExtension/Adguard-Config.xcconfig:2]: This link doesn't work 👇

File: AdguardExtension/Adguard.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 2]
```diff
@@ -30,7 +30,7 @@
       },
       {
         "package": "AGDnsProxy",
-        "repositoryURL": "https://github.com/AdguardTeam/DnsLibs",
+        "repositoryURL": "https://github.com/VisafeTeam/DnsLibs",
```
Comment (vanthoainguyen) [inline AdguardExtension/Adguard.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:33]: You may have to clone this repo and clear all history as well 😂

File: AdguardExtension/Adguard-Config.xcconfig (hunk 1 above)
Comment (huynhthaihoa) [reply AdguardExtension/Adguard-Config.xcconfig:2]: Please update the link, I think it should be: https://github.com/Visafe/VisafeIOS

**Comment by VisafeTeam** [comment]
This pull request is closed since the new code makes the app crash.

**Comment by tohruniverse** [comment]
App bị crash là do replace all trong đó có path nên module sẽ load không được. Ăn trộm thì làm cho trót team Visafe thử update PR đổi toàn bộ tên thư mục xem sao, sure kèo chạy ngon. Thêm nữa là để repo private, làm thêm con VPN nữa forward qua đó rồi tất cả request cứ ném cho bên AdGuard xử lí là đẹp.

> This pull request is closed since the new code makes the app crash.

**Pull request closed without merging**